tests/unit/core/reducers/test_addons.js

```diff
@@ -34,13 +42,14 @@ describe('addon reducer', () => {
         files: [fileOne, fileTwo],
```

Comment:
It would be nice if `addon` here extended `fakeAddon` just to future proof the properties:
```js
const addon = {
  ...fakeAddon,
  slug: 'installable',
  current_version: {
    ...fakeAddon.current_version,
    files: [fileOne, fileTwo],
  },
};
```